### PR TITLE
added registration

### PR DESCRIPTION
**Pull request by MaksymMatuliak**

### CODE REVIEW

File: src/main/java/mate/academy/internetshop/controllers/UserController.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.List;
+import mate.academy.internetshop.lib.Injector;
+import mate.academy.internetshop.model.User;
+import mate.academy.internetshop.service.UserService;
+
+public class UserController extends HttpServlet {
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/internetshop/controllers/UserController.java]: let's add verbs to Controller's names for example  GetAllUsersController (here and everywhere)

File: src/main/java/mate/academy/internetshop/controllers/BuyProductController.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+import mate.academy.internetshop.db.Storage;
+import mate.academy.internetshop.lib.Injector;
+import mate.academy.internetshop.model.Bucket;
+import mate.academy.internetshop.model.Product;
+import mate.academy.internetshop.model.User;
+import mate.academy.internetshop.service.BucketService;
+import mate.academy.internetshop.service.ProductService;
+
+public class BuyProductController extends HttpServlet {
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/internetshop/controllers/BuyProductController.java]:
```suggestion
public class AddProductToShoppingCartController extends HttpServlet {
```

File: src/main/java/mate/academy/internetshop/controllers/CreateProductController.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import java.math.BigDecimal;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.internetshop.lib.Injector;
+import mate.academy.internetshop.model.Product;
+import mate.academy.internetshop.service.ProductService;
+
+public class CreateProductController extends HttpServlet {
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/internetshop/controllers/CreateProductController.java]: AddProductController

File: src/main/webapp/web.xml [hunk 4]
```diff
@@ -9,6 +9,69 @@
...
+        <url-pattern>/buyProduct</url-pattern>
+    </servlet-mapping>
+
+    <servlet>
+        <servlet-name>bucket</servlet-name>
+        <servlet-class>mate.academy.internetshop.controllers.BucketController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>bucket</servlet-name>
+        <url-pattern>/bucket</url-pattern>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/web.xml]: let's use full name of controllers as urls (here and everywhere)

File: src/main/java/mate/academy/internetshop/controllers/UserController.java (hunk 1 above)
Comment (MaksymMatuliak) [reply src/main/java/mate/academy/internetshop/controllers/UserController.java]: done

File: src/main/java/mate/academy/internetshop/controllers/CreateProductController.java (hunk 3 above)
Comment (MaksymMatuliak) [reply src/main/java/mate/academy/internetshop/controllers/CreateProductController.java]: done

File: src/main/webapp/web.xml (hunk 4 above)
Comment (MaksymMatuliak) [reply src/main/webapp/web.xml]: What do you mean?

File: src/main/java/mate/academy/internetshop/controllers/BuyProductController.java (hunk 2 above)
Comment (MaksymMatuliak) [reply src/main/java/mate/academy/internetshop/controllers/BuyProductController.java]: done

File: src/main/java/mate/academy/internetshop/controllers/AddProductToShoppingCartController.java [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import mate.academy.internetshop.lib.Injector;
+import mate.academy.internetshop.service.ShoppingCartService;
+import mate.academy.internetshop.service.ProductService;
+
+public class AddProductToShoppingCartController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate.academy");
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/internetshop/controllers/AddProductToShoppingCartController.java]:
```suggestion
    private static final Injector INJECTOR = Injector.getInstance("mate.academy");
```

Comment (dmytro-hryhoruk) [reply src/main/java/mate/academy/internetshop/controllers/AddProductToShoppingCartController.java]: here and everywhere

Comment (MaksymMatuliak) [reply src/main/java/mate/academy/internetshop/controllers/AddProductToShoppingCartController.java]: done

**Review by dmytro-hryhoruk — COMMENTED** [review]
let's add configure travis for this project https://www.youtube.com/watch?v=BHSsWTbuow4

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**